src/MaterialDesignThemes.Wpf/Converters/SliderToolTipConverter.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace MaterialDesignThemes.Wpf.Converters;
 
-internal class SliderToolTipConverter : IMultiValueConverter
+public class SliderToolTipConverter : IMultiValueConverter
```

Comment:
I propose moving this inside of a new `MaterialDesignThemes.Wpf.Converters.Internal` namespace.